src/components/label-pill/LabelPill.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+import './LabelPill.scss';
+
+export enum LabelPillStatus {
+    DEFAULT = 'default',
+    INFO = 'info',
+    HIGHLIGHT = 'highlight',
+    SUCCESS = 'success',
+    WARNING = 'warning',
+    ALERT = 'alert',
+    ERROR = 'error',
+}
+
+export interface LabelPillProps {
+    children: React.ReactNode;
+    type?: LabelPillStatus;
+}
+
+const LabelPill = React.forwardRef((props: LabelPillProps, ref: React.Ref<HTMLSpanElement>) => {
+    const { children, type = LabelPillStatus.DEFAULT, ...rest } = props;
+    const labelPillClasses = classNames('bdl-LabelPill', {
+        [`bdl-LabelPill--${type}`]: type !== LabelPillStatus.DEFAULT,
```

Comment:
You might be able to get away with just not having this condition, as in, just always add `bdl-LabelPill--${type}`. Having a default class that does nothing is a noop. And helps parent to override for whatever reason.

Reply:
i actually prefer this, as it allows for a classless default look and feel. the current approach only adds a modifier when the modifier will have an effect on the layout.

Reply:
You can always move the defaults into the default modifier